package.json

```diff
@@ -41,6 +41,7 @@
     "pg": "^6.2.4",
     "request": "^2.79.0",
     "sequelize": "4.0.0-2",
+    "sequelize-cli": "^3.0.0",
```

Comment:
`sequelize-cli` doesn't belong in here. This needs to be removed and kept as a local-to-you global NPM package since you're the "only one" as far as the app is concerned who really needed it to generate a migration file.